CHANGELOG.md

```diff
@@ -17,6 +16,9 @@
 - Added support for Configuring the account as a Delegator.
 - A new native dependency has been introduced in order to support the new transaction types mentioned above,
   As a result native dependencies need to be rebuilt before building the Java project.
+- Added initial support for GRPC V2
+- Added initial support for GRPC V2 query GetAnonymityRevokers
+- Added CLI example for GRPC V2 query GetAnonymityRevokers
```

Comment:
These changes should be put under "unreleased changes".

Reply:
fixed